Atomic/util.py

```diff
@@ -43,6 +43,23 @@ def _decompose(compound_name):
         repo, tag = repo.rsplit(':', 1)
     return reg, repo, tag
 
+def atomic_pull_image_by_name(img_name):
+    """
+    Pulls img-name image with atomic. Returns 1 if successful, 0 otherwise.
+    """
+
+    print "Pulling image %s" % img_name
+    try:
```

Comment:
If you really feel the debug output would be beneficial to others, put a if self.debug in front of it.  Either way, then use util.write_out to spill it to stdout.